app.py

```diff
@@ -37,7 +38,7 @@ def run_game(layout: Layout, game_resources: GameResources) -> Panel:
 
 def main() -> None:
     """Main function that sets up game and runs main game loop"""
-    game_resources = GameResources()
+    game_resources = GameResources(testing)
```

Comment:
Do we plan to remove the testing flag at the end of when we're done with the final product?